alpha/apps/kaltura/lib/myPackagerUtils.class.php

```diff
@@ -333,28 +335,60 @@ protected static function captureMappedThumb($entry, $flavorAsset, $capturedThum
 	 * @return bool|mixed
 	 * @throws Exception
 	 */
-	protected static function captureLiveThumb(entry $entry, $liveType, $destThumbPath, $calc_vid_sec, $width = null, $height = null)
+	protected static function captureLiveThumb(entry $entry, $destThumbPath, $calc_vid_sec, $width = null, $height = null)
 	{
-		$packagerCaptureUrl = self::getPackagerUrlFromConf(kPackagerUrlType::LOCAL_LIVE_THUMB);
-		if (!$packagerCaptureUrl)
+		$result = false;
```

Comment:
as we don't assign result a value let not define it but just return the bool value when needed